### PR TITLE
[DR-1225] - Add /status/plan endpoint in the App

### PR DESCRIPTION
**Pull request by nreal**
Hi Team,
The My Consumption and Plan Renewal fields in My plan section through a new endpoint GET `account/status/plan`
Example:

Now, the hour is displayed in PLAN RENEWAL

Could you review?

### CODE REVIEW

File: src/wwwroot/partials/settings/my-plan.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
       </div>
       <div class="flex two flex--wrap">
         <label class="width--full special uppercase">{{ 'my_plan_renewal_date' | translate }}</label>
-        <p class="renewal-date">{{vm.resetDate}}</p>
+        <p class="renewal-date">{{ vm.resetDate | amUtc | amDateFormat:'YYYY-MM-DD HH:mm' }}</p>
```
Comment (andresmoschini) [inline src/wwwroot/partials/settings/my-plan.html]: Nice! what do you think about to be explicit about the timezone?

**Comment by andresmoschini** [comment]
👍

File: src/wwwroot/partials/settings/my-plan.html (hunk 1 above)
Comment (nreal) [inline src/wwwroot/partials/settings/my-plan.html]: Yes, done

**Comment by andresmoschini** [comment]
👍

**Comment by mvillaseco** [comment]
👍

**Pull request merged**